src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -273,6 +274,7 @@ public IEnumerable<PackageItemViewModel> GetCurrent()
                 if (listItemViewModels.TryGetValue(packageId, out PackageItemViewModel existingListItem))
                 {
                     existingListItem.InstalledVersions.Add(packageVersion);
+                    existingListItem.UpdateInstalledPackagesVulnerabilities(new PackageIdentity(packageId, packageVersion));
```

Comment:
Can we just use the `metadataContextInfo.Identity` instead of creating and instantiating a new one?